### PR TITLE
@86epq019c #ForgotPassword Page UI

### PR DESCRIPTION
**Pull request by LizterBabael**

### CODE REVIEW

File: apps/web-customer/src/core/Schema/ForgotPasswordValidation.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import * as yup from "yup";
+
+export const forgotPasswordSchema = yup.object({
+  email: yup.string().required("Email is required").default(""),
```
Comment (be3jaay) [inline apps/web-customer/src/core/Schema/ForgotPasswordValidation.ts:4]: should be email

File: apps/web-customer/src/pages/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { ForgotPasswordFormBlock } from "@/components/blocks/ForgotPasaswordBlock/ForgotPasswordBlock";
+
+export const ForgotPasswordPage = () => {
```
Comment (be3jaay) [inline apps/web-customer/src/pages/ForgotPassword.tsx:3]: should be similar to the file name.

**Review by be3jaay — APPROVED** [review]

File: apps/web-customer/src/components/blocks/ForgotPasaswordBlock/ForgotPasswordBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { ForgotPasswordForm } from "./ForgotPasswordForm";
+import {
+  forgotPasswordSchema,
+  forgotPasswordType,
+} from "@/core/Schema/ForgotPasswordValidation";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+
+export function ForgotPasswordFormBlock() {
+  const form = useForm({
```
Comment (DropitupQwerty) [inline apps/web-customer/src/components/blocks/ForgotPasaswordBlock/ForgotPasswordBlock.tsx:10]: Move this inside the block

**Review by JezChristian — APPROVED** [review]

**Review by acostales08 — APPROVED** [review]

**Review by JMSevilla — CHANGES_REQUESTED** [review]
This implementation is not aligned with our current system diagram.

**Review by JMSevilla — APPROVED** [review]

**Pull request merged**